TheMET/TheMET/FeaturedArtistsContentView/FeaturedArtistsContentView.swift

```diff
@@ -81,11 +81,11 @@ class FeaturedArtistsContentView: UIView, UICollectionViewDelegate, UICollection
                     if let image = image {
                         cell.backgroundState = .loaded(image)
                     } else {
-                        cell.backgroundState = .failed
+                        cell.backgroundState = .failed(.noInternet)
```

Comment:
This state loading is failed not because there is no internet. The image loading did not returned an image.